### DISCUSSION TITLE
Story Agent项目

### DISCUSSION THREAD

**Discussion by cgwyx** [modelscope/ms-agent#376] [Q&A]
Story Agent项目https://www.modelscope.cn/studios/iic/story_agent/summary， 文生图AI-ModelScope/stable-diffusion-xl-base-1.0模型已下载到本地，如何实现本地调用_local_call，
需要修改这个文件配置\modelscope-agent\demo\story_agent\cfg_tool_template.json吗？这样修改，也无法调用
    "modelscope_image-generation": {
        "is_active": true,
        "use": true,
        "is_remote_tool": false,
        "pipeline_params": {
		    "use_safetensors": true
        }

**Comment by zzhangpurdue** [comment]
这个方法也可以用，但是需要本地配置 MODELSCOPE_API_TOKEN
相关获取地址在  https://modelscope.cn/my/myaccesstoken  登陆获取
